deploy.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+
+# run this script as normal user w/o sudo from within the gin-repo root folder.
```

Comment:
Its kind of strange to me that it advises non sudo this while sudoing with a prompt below. Would probably surprise me if i where a naive deployer. I guess i get the why and how still...